lib/ueberauth/strategy/slack.ex

```diff
@@ -204,6 +207,13 @@ defmodule Ueberauth.Strategy.Slack do
     |> List.first
   end
 
+  def get_config(key) do
```

Comment:
The options function will pull these out of the config. for example:

    option(conn, :default_scope)